src/symbol_mt5.py

```diff
@@ -43,8 +43,10 @@ def get_candlestick_time(self) -> int:
         return rounded_candlestick_time
     
     def get_symbol_info(self) -> bool:
-        symbol_info = mt5.symbol_info(self.symbol)._asdict()
-        return symbol_info
+        symbol_info = mt5.symbol_info(self.symbol_name)
+        if symbol_info == None:
```

Comment:
think we should use 'is' instead of '=='
https://sourcery.ai/blog/python-pandas-compare-to-none/#:~:text=PEP%208%20has%20a%20clear,could%20lead%20to%20unexpected%20results.